lib/grinder_sdk.dart

```diff
@@ -44,7 +44,7 @@ class Dart {
   /// Returns the stdout.
   static String run(String script,
       {List<String> arguments : const [], bool quiet: false,
-       String packageRoot, String workingDirectory, int vmNewGenHeapMB,
+       String packageRoot, RunOptions runOptions, int vmNewGenHeapMB,
```

Comment:
I think just `options` would be clear enough when calling the `run` and `runAsync` method.  Question is whether it would be clear when calling e.g. `Dartfmt.format(options: new RunOptions(...))`.  So I'd say stick with `runOptions` everywhere to be consistent.